src/components/atoms/CheckToDisplayAsFull/CheckToDisplayAsFull.tsx

```diff
@@ -27,34 +29,20 @@ const CheckToDisplayAsFull = ({
     return 'Full';
   }, [spaceLeft, capacity]);
 
-  const spaceAvailable = spaceLeft && spaceLeft < 0;
+  const spaceAvailable = spaceLeft === 0;
```

Comment:
I think the name of the varible is quite confusing, surely if spaces left === 0, then it's not available?

Reply:
Removed spaceAvailable for just spaceLeft now